app/components/schools/mentors/summary_component.rb

```diff
@@ -0,0 +1,50 @@
+module Schools
+  module Mentors
+    class SummaryComponent < ViewComponent::Base
+      include TeacherHelper
+
+      def initialize(mentor:, school:)
+        @mentor = mentor
+        @school = school
+      end
+
+      def call
+        govuk_summary_card(title: link_to_mentor(@mentor)) do |card|
+          card.with_summary_list(
+            classes: %w[govuk-summary-list--no-border],
+            rows: [trn_row, assigned_ects_row]
+          )
+        end
+      end
+
+    private
+
+      def link_to_mentor(mentor)
```

Comment:
It is possible to remove the param and access it from the instance variable

Reply:
It is indeed and good shout again! I've updated here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/476/commits/16c9664e18066411c5f0591d2078913a84813033